gradle.properties

```
@@ -11,61 +11,61 @@ slf4jVersion=1.7.30
+stdlibJavaArraysVersion=1.3.0-20230620-192700-c836fd3
 stdlibRegexVersion=1.4.3
-stdlibTimeVersion=2.2.4
-stdlibUrlVersion=2.2.3
-stdlibXmldataVersion=2.5.0
-observeVersion=1.0.7
+stdlibTimeVersion=2.3.0-20230620-192500-12d3432
+stdlibUrlVersion=2.3.0-20230620-193100-90b6489
+stdlibXmldataVersion=2.6.0-20230620-192500-909c3b3
+observeVersion=1.1.0-20230620-193900-57e0c73
 stdlibMathVectorVersion=1.0.1
 
 
 # Stdlib Level 02
-stdlibCryptoVersion=2.3.1
-stdlibLogVersion=2.7.1
-stdlibOsVersion=1.6.0
-stdlibProtobufVersion=1.4.0
-stdlibRandomVersion=1.3.1
-stdlibTaskVersion=2.3.2
-stdlibXsltVersion=2.4.0
-observeInternalVersion=1.0.6
+stdlibCryptoVersion=2.4.0-20230620-202300-53ac3fb
+stdlibLogVersion=2.8.0-20230620-203000-74034d2
+stdlibOsVersion=1.7.0-20230620-202800-f75dd63
+stdlibProtobufVersion=1.5.0-20230620-202600-8d3ad44
+stdlibRandomVersion=1.4.0-20230620-162300-6c8d20e
+stdlibTaskVersion=2.4.0-20230620-204200-bd8329a
+stdlibXsltVersion=2.5.0-20230620-202700-3ea55e6
+observeInternalVersion=1.1.0-20230620-201800-18bbd1b
 
 # Stdlib Level 03
-stdlibCacheVersion=3.5.0
-stdlibFileVersion=1.7.1
-stdlibFtpVersion=2.7.1
-stdlibMimeVersion=2.7.1
-stdlibTcpVersion=1.7.1
-stdlibUdpVersion=1.7.1
-stdlibUuidVersion=1.5.1
+stdlibCacheVersion=3.6.0-20230620-211300-8e2e2da
+stdlibFileVersion=1.8.0-20230620-210800-e8f8123
+stdlibFtpVersion=2.8.0-20230620-214200-1f1f3d2
+stdlibMimeVersion=2.8.0-20230620-205000-a1e0453
+stdlibTcpVersion=1.8.0-20230620-205400-e092f0f
+stdlibUdpVersion=1.8.0-20230620-205000-38f5859
+stdlibUuidVersion=1.6.0-20230620-211300-6fdbff8
 
 # Stdlib Level 04
-stdlibAuthVersion=2.8.0
-stdlibEmailVersion=2.7.1
-stdlibJwtVersion=2.8.0
-stdlibOAuth2Version=2.8.0
-stdlibYamlVersion=0.3.1-20230612-141800-3f686c6
-stdlibTomlVersion=0.3.1-20230609-180200-0462a55
+stdlibAuthVersion=2.9.0-20230620-221100-6b88179
+stdlibEmailVersion=2.8.0-20230620-221300-6012f69
+stdlibJwtVersion=2.9.0-20230620-221100-adcdde4
+stdlibOAuth2Version=2.9.0-20230620-214300-9df2b5a
+stdlibYamlVersion=0.4.0-20230621-095100-aa7f98e
+stdlibTomlVersion=0.4.0-20230621-093200-a851275
 stdlibEdiVersion=1.0.1
 
 # Stdlib Level 05
-stdlibHttpVersion=2.8.0
+stdlibHttpVersion=2.9.0-20230621-065800-bb69fb5
 
 # Stdlib Level 06
-stdlibGrpcVersion=1.8.0
-stdlibTransactionVersion=1.6.0
-stdlibWebsocketVersion=2.8.0
-stdlibWebsubVersion=2.8.0
-stdlibWebsubhubVersion=1.8.0
+stdlibGrpcVersion=1.9.0-20230621-101400-b89816c
+stdlibTransactionVersion=1.7.0-20230621-075800-07ccae7
+stdlibWebsocketVersion=2.9.0-20230621-092700-d2720de
+stdlibWebsubVersion=2.9.0-20230621-094800-643bba5
+stdlibWebsubhubVersion=1.9.0-20230621-094900-f1b0bad
 
 # Stdlib Level 07
-stdlibGraphqlVersion=1.8.0
-stdlibSqlVersion=1.9.0
+stdlibGraphqlVersion=1.9.0-20230621-103200-0f3ae1f
+stdlibSqlVersion=1.10.0-20230621-085100-7175ed0
 
 # Stdlib Level 08
-stdlibPersistVersion=1.0.0
+stdlibPersistVersion=1.1.0-20230620-202500-81fc407
 
 # Persist Tool
 persistToolVersion=1.0.0
```

Persist-tool time stamp pack is released now. 
```suggestion
persistToolVersion=1.1.0-20230621-124200-7e5f605
```

---

Updated.